lib/git_worktree.rb

```diff
@@ -258,6 +258,14 @@ def checkout
     end
   end
 
+  def check_connection(url)
```

Comment:
I know this is closed, but just as a note for me so I don't lose context, we were in the process of replacing this method with `Net::Ping::HTTP.new(url).ping?` when we hit the subsequent error about the job not existing.